vochain/indexer/indexertypes/types.go

```diff
@@ -336,6 +336,17 @@ func (b *BlockMetadata) String() string {
 	return builder.String()
 }
 
+// TokenTransferMeta contains the information of a token transfer and some extra useful information.
+// The types are compatible with the SQL defined schema.
+type TokenTransferMeta struct {
+	Amount    uint64          `json:"amount"`
+	From      types.AccountID `json:"from"`
```

Comment:
If this needs to be JSON encoded I think we should add a Method on types.AccountID to Marshal the []byte address correctly (in hex format). Else declare it as `type.HexBytes`